### PR TITLE
Jest testing and initial snapshot test.

### PR DESCRIPTION
**Pull request by weerd**
This PR adds [Jest](https://facebook.github.io/jest/) for testing our JS code and specifically adds an initial snapshot test.

Run tests using:

```shell
$ npm run test
```

or just use the shortcut of `npm t`.

### CODE REVIEW

**Comment by weerd** [comment]
Fixing some ESLint errors. Hold please!

**Comment by DFurnes** [comment]
Woop woop! 👏 Let's add this to our Wercker builds as well!

**Comment by weerd** [comment]
oh jes(t)! will do.

File: package.json [hunk 1]
```diff
@@ -41,9 +49,17 @@
     "@dosomething/eslint-config": "^2.0.0",
     "@dosomething/webpack-config": "^2.0.0",
     "babel-eslint": "^7.2.2",
+    "babel-jest": "^19.0.0",
     "dotenv": "^4.0.0",
+    "enzyme": "^2.8.2",
+    "enzyme-to-json": "^1.5.1",
     "eslint-loader": "^1.7.1",
+    "extract-text-webpack-plugin": "^1.0.1",
```
Comment (DFurnes) [inline package.json]: Get this outta here!

**Pull request merged**